packages/@expo/use-updates/src/UseUpdates.types.ts

```diff
@@ -0,0 +1,157 @@
+import Constants from 'expo-constants';
+import type { UpdatesLogEntry } from 'expo-updates';
+
+export type ClassicManifest = NonNullable<typeof Constants.manifest>;
+
+export type Manifest = ClassicManifest | NonNullable<typeof Constants.manifest2>;
+
+/**
+ * Structure encapsulating information on the currently running app
+ * (either the embedded bundle or a downloaded update).
+ */
+export type CurrentlyRunningInfo = {
+  /**
+   * The UUID that uniquely identifies the currently running update if `expo-updates` is enabled. The
+   * UUID is represented in its canonical string form (`xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`) and
+   * will always use lowercase letters. In development mode, or any other environment in which
+   * `expo-updates` is disabled, this value is `null`.
+   */
+  updateId: string | null;
+  /**
+   * The channel name of the current build, if configured for use with EAS Update. Null otherwise.
+   */
+  channel: string | null;
+  /**
+   * If `expo-updates` is enabled, this is a `Date` object representing the creation time of the update
+   * that's currently running (whether it was embedded or downloaded at runtime).
+   *
+   * In development mode, or any other environment in which `expo-updates` is disabled, this value is
+   * null.
```

Comment:
```suggestion
   * `null`.
```

Inconsistent use of representing null value with or without syntax highlights. For example, in line 17, we are using inline code syntax. In general, for consistency, we should prefer sticking to one pattern throughout.